Created Devzat status page (#193)
quackduck/devzat: README.md

Feel free to make a [new issue](https://github.com/quackduck/devzat/issues) if something doesn't work.

> If you think the main devzat server is down (devzat.hackclub.com), you can check [Our Uptimebot Site](https://stats.uptimerobot.com/kxMQqfYk4y) for more information.

Reviewer: ```suggestion
See the [status site](https://stats.uptimerobot.com/kxMQqfYk4y) of the main Devzat server to check if it might be down.
```
